### PR TITLE
(#3548) Replace Acquia Pipelines w/ Github Actions

### PR DESCRIPTION
**Pull request by bryanpizzillo**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+          #blt tests --define drush.alias='${drush.aliases.ci}' -D behat.web-driver=chrome --no-interaction --ansi --verbose
+
+          ## Behat tests will run the server
+          blt tests:behat:run --define drush.alias='${drush.aliases.ci}' --environment=ci --define tests.run-server=true -D behat.web-driver=chrome --ansi --verbose
+
+          ## PHP Unit tests do not start the server
+          ##blt tests:server:start --define drush.alias='${drush.aliases.ci}' --environment=ci --define tests.run-server=true -D behat.web-driver=chrome --ansi --verbose
+          drush rs 0.0.0.0:8888 &
+          blt custom:paratest  --define drush.alias='${drush.aliases.ci}' --environment=ci --define tests.run-server=true -D behat.web-driver=chrome --ansi --verbose
+          blt tests:server:kill --define drush.alias='${drush.aliases.ci}' --environment=ci --define tests.run-server=true -D behat.web-driver=chrome --ansi --verbose
```
Comment (jfrank-nih) [inline .github/workflows/ci.yml:129]: Is this required since the server:start line is canceled out?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+          ## We don't set git info for right now as we are in the NCIOCPL origin right now.
+      - name: Validate
+        run: |
+          blt validate --define drush.alias='${drush.aliases.ci}' --ansi --verbose --no-interaction
+      - name: Setup App
+        run: |
+          ## The setup app script cannot create build.json as we are in the NCIOCPL origin.
+          export PHANTOMJS_CDNURL=https://npm.taobao.org/mirrors/phantomjs
+          blt setup --define drush.alias='${drush.aliases.ci}' --environment=ci --no-interaction --ansi --verbose
+      - name: Test
```
Comment (jfrank-nih) [inline .github/workflows/ci.yml:113]: Splitting these apart into individual test steps could help (slightly) in tracking them.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bryanpizzillo) [reply .github/workflows/ci.yml:131]: Technically tests:server:kill kills the drush rs process that was backgrounded. I have removed the commented out scripts.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (bryanpizzillo) [reply .github/workflows/ci.yml]: I split apart the tests into Behat and PHPUnit.

**Review by jfrank-nih — APPROVED** [review]

**Pull request merged**